sync_chart_version.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+tagVersion=""
+if [ ${GITHUB_REF_TYPE} == "tag" ]
+then
+    tagVersion=${GITHUB_REF_NAME}
+fi
+export CHART_VERSION=$(expr ${tagVersion:1})
+
+echo "${GITHUB_REF}"
+echo "${GITHUB_REPOSITORY}"
+echo "${GITHUB_ACTOR}"
```

Comment:
Is this really needed? Does this has any sensitive information.

Reply:
we have these echo statements in update_gh_pages.sh as well
this helps debugging which repo its pulling and actor name who is committing